### PR TITLE
Change intro text

### PR DESCRIPTION
**Pull request by EstebanDalelR**
## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)  
- [ ] New feature (non-breaking change which adds functionality)  
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)  
- [x] Documentation update  
- [ ] Chore: cleanup/renaming, etc  
- [ ] RFC  
## Acceptance

- [x] I have read [the Contributing guidelines](CONTRIBUTING.md)
- [x] I have read the [Code of Conduct](CODE_OF_CONDUCT.md)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,7 +9,7 @@
 [![Rating](https://img.shields.io/visual-studio-marketplace/r/WatermelonTools.watermelon-tools?style=flat-square)](https://marketplace.visualstudio.com/items?itemName=WatermelonTools.watermelon-tools&ssr=false#review-details)
 [![Discord](https://img.shields.io/discord/933846506438541492?style=flat-square)](discord.gg/xNDFXx9447)
 
-Watermelon is an **open-source** integration between **GitHub** and **Visual Studio Code** to help developers go beyond Git Blame. Watermelon makes you an expert on any file instantly by running `git blame` for you and telling you why a block of code was written that way by someone else.
+Watermelon is an **open-source** for **Visual Studio Code** to help developers understand code better. Watermelon makes you an expert on any file by selecting code and bringing all the information available.
```
Comment (baristaGeek) [inline README.md:12]: open-source integration for Visual Studio Code

Comment (baristaGeek) [reply README.md:12]: Mention "your code archeology toolbox" somewhere, pls

**Review by baristaGeek — CHANGES_REQUESTED** [review]
Pls

File: README.md (hunk 1 above)
Comment (EstebanDalelR) [reply README.md]: I merged the three texts.

**Review by baristaGeek — APPROVED** [review]

**Pull request merged**